src/types/user.types.ts

```diff
@@ -36,6 +47,31 @@ export const userResponse = Type.Object({
 
 export type UserResponse = Static<typeof userResponse>;
 
+export const createUserRequestBody = Type.Object({
+  email: Type.String(),
+  forename: Type.String(),
+  surname: Type.String(),
+  password: Type.String(),
+});
+
+export type CreateUserRequestBody = Static<typeof createUserRequestBody>;
+
+export type CreateUserRequest = FastifyRequest<{
+  Reply: UserResponse;
+  Body: CreateUserRequestBody;
+}>;
+
+interface IUserResponse extends RouteGenericInterface {
```

Comment:
Käytetään mielummin typejä interfacejen sijasta

```suggestion
type UserResponse = RouteGenericInterface & {...}
```

Reply:
Voidaan toki väitellä aiheesta

https://medium.com/swlh/prefer-interfaces-4fb72c25b4c

Reply:
Jos päädyt interfaceihin yli typejen niin laita samalla eslintin sääntö interfaces over types voimaan

Reply:
kts: https://github.com/Tampereen-Urheilusukeltajat/Blenderi-Backend/pull/6#discussion_r992645527